spec/components/sdg/goals/target_component_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+describe SDG::Goals::TargetComponent, type: :component do
+  let(:targets) { SDG::Goal[1].targets }
+  let(:component) { SDG::Goals::TargetComponent.new(targets) }
+
+  before do
+    Setting["feature.sdg"] = true
+  end
+
+  it "does not render when the feature is disabled" do
+    Setting["feature.sdg"] = false
+
+    render_inline component
+
+    expect(page).not_to have_css "#tab_global_targets"
+    expect(page).not_to have_css ".global-targets"
+  end
+
+  it "renders tabs panel" do
+    render_inline component
+
+    expect(page).to have_css "#tab_global_targets"
+    expect(page).to have_css ".global-targets"
+  end
+
+  it "renders code and title for each target" do
+    render_inline component
+
+    expect(page).to have_content "1.1"
+    expect(page).to have_content "By 2030, eradicate extreme poverty for all people everywhere, currently measured as people living on less than $1.25 a day"
```

Comment:
This proves changing line length limit to 140 will only result in requests to change it to 160 (only way this line wouldn't be an offense) :smirk:.

```suggestion
    expect(page).to have_content "By 2030, eradicate extreme poverty for all people everywhere"
```